### PR TITLE
Add various augmentations to the feature vector

### PR DESCRIPTION
**Pull request by hdefazio**
## What does this PR do?

Adds the following augmentations
1 -  adjust the center points of all objects by deltas for hands/objects  (aka update the calculated distances)
2 - Normalize the pixel values by the frame size
3 - adjust the activation feature values by a delta 

Description of the feature vector (version 2) can be found: 
https://github.com/PTG-Kitware/angel_system/blob/4180e66671ae11a9d4b5224533fb8a57ab47b3b7/angel_system/impls/detect_activities/detections_to_activities/utils.py#L46

## Before submitting

- [ ] Did you make sure **title is self-explanatory** and **the description concisely explains the PR**?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **test your PR locally** with `pytest` command?
- [ ] Did you **run pre-commit hooks** with `pre-commit run -a` command?

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: src/data/components/augmentations.py [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        """
+        :param w: Width of the frames
+        :param h: Height of the frames
+        :param num_obj_classes: Number of object classes from the detections
+            used to generate the features
+        :param feat_version: Algorithm version used to generate the input features
+        """
+        super().__init__()
+
+        self.im_w = im_w
```
Comment (cfunk1210) [inline src/data/components/augmentations.py:191]: The features seem larger than the im height and width:

self.feature_frames[:,:82].min(axis=0)array([ -858.2777 , -2453.1472 , -1371.659  , -1570.6676 , -1726.0548 ,       -1191.5568 , -1246.9102 , -1164.7008 , -1786.2899 , -1837.5206 ,       -1863.5317 , -1835.0763 , -1325.7748 , -1275.6404 , -3080.0647 ,       -1294.1636 , -2094.3445 , -2477.2158 , -1363.6292 , -1522.2816 ,       -1632.206  , -1453.6261 , -1272.253  , -1797.5276 , -1254.3243 ,       -1649.124  ,  -999.3401 , -1140.8475 , -1149.4592 , -1068.3418 ,       -1058.5824 ,  -973.12225, -1314.1495 , -1361.6561 , -1305.8376 ,       -1341.553  , -1274.2898 , -1586.8881 , -1103.4714 ,  -980.2082 ,       -1066.6299 , -1352.585  , -1443.6705 , -1129.6648 , -1736.9276 ,       -1796.9462 , -1587.1392 , -1861.2399 , -1866.9806 , -1246.3413 ,       -1639.9084 , -1875.7797 , -1442.594  , -1123.2693 , -1360.2676 ,       -1255.9332 , -1079.3168 , -1562.4827 , -1424.884  , -1361.5192 ,       -1395.2253 , -1889.3933 , -1127.5387 , -1868.5323 , -1308.3363 ,       -1131.8317 , -1248.6018 , -1367.6566 , -1129.9913 , -1229.109  ,       -1215.9607 , -1084.3501 ,  -946.93115, -1072.498  , -1306.0743 ,       -1191.2345 , -1557.0569 , -1853.8293 , -1340.5442 , -1378.9635 ,       -2181.1196 , -1324.466  ], dtype=float32)

File: src/data/components/augmentations.py [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+        return delta
+
+    def forward(self, window):
+        delta = self.init_delta()
+        
+        if self.feat_version == 1:
+            window[:] = np.where(
+                window[:] != 0,
+                np.clip(window[:] * delta, 0, 1),
```
Comment (cfunk1210) [inline src/data/components/augmentations.py]: Multiplying to move the points would make things farther from center even farther away.  I think this needs to be an additive change.

**Review by cfunk1210 — COMMENTED** [review]
There are some issues unresolved but I did a commit to fix a few of them

File: src/data/components/augmentations.py (hunk 1 above)
Comment (hdefazio) [reply src/data/components/augmentations.py:191]: resolved - the hololens hand bboxes may be outside of the image size

**Pull request merged**